clients/src/main/java/org/apache/kafka/clients/admin/GroupMember.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.clients.admin;
+
+import org.apache.kafka.common.message.LeaveGroupRequestData.MemberIdentity;
+
+import java.util.Objects;
+
+/**
+ * A struct containing member's info.
+ */
+public class GroupMember {
+    public String memberId;
+    public String groupInstanceId;
+
+    public GroupMember(String memberId, String groupInstanceId) {
+        this.memberId = memberId;
+        this.groupInstanceId = groupInstanceId;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (o instanceof GroupMember) {
+            GroupMember otherMember = (GroupMember) o;
+            return this.memberId.equals(otherMember.memberId) &&
+                       this.groupInstanceId.equals(otherMember.groupInstanceId);
+        } else {
+            return false;
+        }
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(groupInstanceId, memberId);
+    }
+
+    MemberIdentity getMemberIdentity() {
```

Comment:
nit: maybe `toMemberIdentity` so that it is clear this is a converter and not an accessor?